CRM/Case/XMLProcessor/Process.php

```diff
@@ -88,7 +88,7 @@ public function process($xml, &$params) {
       // create relationships for the ones that are required
       foreach ($xml->CaseRoles as $caseRoleXML) {
         foreach ($caseRoleXML->RelationshipType as $relationshipTypeXML) {
-          if ((int ) $relationshipTypeXML->creator == 1) {
+          if ($relationshipTypeXML->creator) {
             if (!$this->createRelationships($relationshipTypeXML,
               $params
             )
```

Comment:
Boy you could really get sucked into making this file better. This conditional has some strange line-breaks that I would get rid of... and the `return FALSE;` below is unreachable after calling `CRM_Core_Error::fatal();`. And of course we should be throwing exceptions instead of using that function anyway... where to stop?

Reply:
I know the feeling. Beer solves it temporarily.

Reply:
Since there were some weird test fails anyway I updated those line breaks too.